src/draive/gemini/lmm_generation.py

```
@@ -127,142 +155,358 @@ async def lmm_completion(  # noqa: C901, PLR0912, PLR0915
+                        yield LMMStreamChunk.of(MultimodalContent.empty, eod=True)
+                    break
+                elif finish_reason == FinishReason.MAX_TOKENS:
+                    raise GeminiException(
+                        "Invalid Gemini completion - exceeded maximum length!",
+                        completion_chunk,
+                    )
+                else:
+                    raise GeminiException(
+                        f"Gemini completion generation failed! Reason: {finish_reason}",
+                        completion_chunk,
+                    )
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "break", remove the leading "el" from "elif"

(R1723)